CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ find_package(catkin REQUIRED
     moveit_ros_planning_interface
     pluginlib
     geometric_shapes
+    moveit_visual_tools
```

Comment:
If you would like to rely on `moveit_visual_tools`, shouldn't we move the package in with the rest of moveit?